WickedEngine/wiXInput.cpp

```diff
@@ -112,9 +112,20 @@ namespace wiXInput
 #else
 namespace wiXInput
 {
-	void Update() {}
-	int GetMaxControllerCount() { return 0; }
-	bool GetControllerState(wiInput::ControllerState* state, int index) { return false; }
-	void SetControllerFeedback(const wiInput::ControllerFeedback& data, int index) {}
+	void Update() {
```

Comment:
XInput will never be on Linux, so these empty functions don't make sense, just remove.